### ISSUE TITLE
Routes on version 5.10.0

### ISSUE THREAD

**Issue by t3asp00n** [Project-OSRM/osrm-backend#4309]
I've been using the local instance of osrm version 5.8.0 for almost 2 months now, so far it has work perfectly. I've now install version 5.10.0 on a new server and some of the routes I got with the previous version are not working anymore, like:

localhost:5000/route/v1/driving/-70.73719024658205,-29.093077058146022;-70.5863,-33.4465?overview=false&alternatives=true&steps=true&hints=;

This returns "Impossible route between points" in version 5.10. Could it be because I'm using an extract from only one country (Chile)? I noticed that in the demo I tried a similar route and the result given was routing me to the neighboring country (Argentina):

![screenshot from 2017-07-19 15-35-16](https://user-images.githubusercontent.com/28709799/28385932-e900fd08-6c97-11e7-987c-badf473cce48.png)

Any help would be greatly appreciated.

**Comment by MoKob** [comment]
@t3asp00n I've looked at the problem and I found there might be some data issues either introduced or now handled with an updated profile.
The issue in question is this way: https://www.openstreetmap.org/way/405703101#map=17/-29.61685/-71.28294 which is marked as in construction in OSM.

The debug map doesn't show it to be present in the routing graph, due to this (see [here](http://map.project-osrm.org/debug/#17.21/-29.61721/-71.28501)).

![screen shot 2017-07-20 at 10 52 03 2](https://user-images.githubusercontent.com/12932279/28409039-b4c3f520-6d39-11e7-9012-b508a3e603fc.png)

You either have to fix the data at that bridge which currently disconnects Chile or use a larger extract, to ensure there are detours which could be taken.
